shared/backend.tfvars

```diff
@@ -0,0 +1,4 @@
+storage_account_name="storage-account-name"
+access_key="storage-account-access-key"
+container_name="storage-account-container"
+key="tfstate-key"
```

Comment:
Perhaps a better default name, such as `cobalt-test-terraform.tfstate` to indicate this is a tfstate file.  Something other than *-key in the name.